web/app.js

```diff
@@ -1329,12 +1329,33 @@ const PDFViewerApplication = {
       pdfViewer.optionalContentConfigPromise.then(optionalContentConfig => {
         this.pdfLayerViewer.render({ optionalContentConfig, pdfDocument });
       });
+      if ("requestIdleCallback" in window) {
+        window.requestIdleCallback(() => {
+          this._collectTelemetry();
+        });
+      }
     });
 
     this._initializePageLabels(pdfDocument);
     this._initializeMetadata(pdfDocument);
   },
 
+  /**
+   * A place to fetch data for telemetry after one page is rendered and the
+   * viewer is idle.
+   */
+  async _collectTelemetry() {
+    if (!this.pdfDocument) {
+      return;
+    }
+    let markInfo = await this.pdfDocument.getMarkInfo();
```

Comment:
This variable and the next one can be `const` (I believe this is also why the CI failed).